### PR TITLE
fix:TOS-775 Step is not getting created when we try creating Element from Test step

### PR DESCRIPTION
**Pull request by manikanta-reddy-t**

### CODE REVIEW

File: ui/src/app/services/chrome-recoder.service.ts [hunk 1]
```diff
@@ -75,7 +75,7 @@ export class ChromeRecorderService {
       this.messageEvent = event;
       if (this.isInstalledEvent) {
         this.isInstalled = true;
-      } else if (this.isStoppedRecorder) {
+      } else if (this.isStoppedRecorder && this.isStepRecording) {
```
Comment (guganabu) [inline ui/src/app/services/chrome-recoder.service.ts:78]: how does the isStoppedRecorder and isStepRecording align same?

Comment (guganabu) [reply ui/src/app/services/chrome-recoder.service.ts:78]: the prop can be named as isStepRecorder | isElementRecorder accordingly based on the context of recorder.

Comment (manikanta-reddy-t) [reply ui/src/app/services/chrome-recoder.service.ts]: updated to isSteprecorder

File: ui/src/app/services/chrome-recoder.service.ts [hunk 2]
```diff
@@ -75,8 +75,8 @@ export class ChromeRecorderService {
       this.messageEvent = event;
       if (this.isInstalledEvent) {
         this.isInstalled = true;
-      } else if (this.isStoppedRecorder) {
-        this.isStepRecording = false;
+      } else if (this.isStoppedRecorder && this.isStepRecorder) {
+        this.isStepRecorder = false;
```
Comment (guganabu) [inline ui/src/app/services/chrome-recoder.service.ts:79]: why do we need to toggle stepRecorder flag? what is the use case here? @manikanta-reddy-t

Comment (manikanta-reddy-t) [reply ui/src/app/services/chrome-recoder.service.ts:79]: i think we will enter into that block only when we stopped recording and if it is an step recorder. when we stopped recording then we will changing that to an false.

File: ui/src/app/components/webcomponents/step-summary.component.ts [hunk 3]
```diff
@@ -93,7 +93,8 @@ export class StepSummaryComponent implements OnInit {
       name: name,
       versionId: this.options?.version?.id,
       testCaseId: this.options?.testCase?.id,
-      isStepRecordView: this.options?.isStepRecordView
+      isStepRecordView: this.options?.isStepRecordView,
+      isNoRefresh:true,
```
Comment (guganabu) [inline ui/src/app/components/webcomponents/step-summary.component.ts:97]: seems unused prop

Comment (manikanta-reddy-t) [reply ui/src/app/components/webcomponents/step-summary.component.ts]: removed

**Review by guganabu — APPROVED** [review]

**Review by guganabu — APPROVED** [review]

**Pull request merged**